pkg/cmd/pipeline/start.go

```diff
@@ -295,7 +295,9 @@ func (opt *startOptions) startPipeline(pipelineStart *v1beta1.Pipeline) error {
 		if err != nil {
 			return err
 		}
-		pr.Spec.Timeouts.Pipeline = &metav1.Duration{Duration: timeoutDuration}
+		pr.Spec.Timeouts = &v1beta1.TimeoutFields{
+			Pipeline: &metav1.Duration{Duration: timeoutDuration},
+		}
```

Comment:
shouldn't we be having `pr.Spec.Timeout = &metav1.Duration{Duration: timeoutDuration}` ? with the removal of `Timeout` field, we can remove the `--timeout` flag as well. What do you think @piyush-garg ?